flow-typed/types/advertising.js

```diff
@@ -9,7 +9,8 @@
 type _KPAdObject = {
   url?: Array<string>,
   response?: Array<string>,
-  bumper?: boolean
+  bumper?: boolean,
+  prebid?: KPPrebidConfig
```

Comment:
as we talked this config should be in the root level. in advertising object.
we can add it in the future also in the ad level, but not for now and some of params are irrelevant in ad level (e.g. libUrl).

Reply:
No, it's on the top level for general config like libUrl and could be for specific config as well if there is only one.
The ad level config is for set specific config per ad like bidder for preroll or midroll and it also allows me to know which ad in the ad layout suppose to use prebid.

Reply:
Maybe the config type should be more strict without libUrl in ad level